cocos/scripting/js-bindings/jswrapper/sm/Object.cpp

```diff
@@ -176,13 +176,13 @@ namespace se {
         return ok;
     }
 
-    void Object::setProperty(const char* name, const Value& v)
+    bool Object::setProperty(const char* name, const Value& v)
     {
         JS::RootedObject object(__cx, _getJSObject());
 
         JS::RootedValue value(__cx);
         internal::seToJsValue(__cx, v, &value);
-        JS_SetProperty(__cx, object, name, value);
+        return JS_SetProperty(__cx, object, name, value);
```

Comment:
se::Object::setProperty should return boolean value to indicate whether the invocation is successful.